docs/functions.rst

```diff
@@ -0,0 +1,98 @@
+Functions in the Tax and Transfer System
+========================================
+
+Allgemein
```

Comment:
I would suggest to not summarize different modules under this general category "Allgemein". Instead more categories, even if they only contain one module.